src/base.ts

```diff
@@ -80,7 +80,8 @@ export default abstract class extends Command {
 
   recorderFromEnv(prefix: string): Recorder {
     let sink: Sink = new DiscardSink();
-    if (process.env.ASYNCAPI_METRICS !== 'false') {
+    
+    if (process.env.ASYNCAPI_METRICS !== 'false' && process.env.CI !== 'true') {
```

Comment:
Shouldn't be this an OR rather than an AND?

```suggestion
    if (process.env.ASYNCAPI_METRICS !== 'false' || process.env.CI !== 'true') {
```

Reply:
Not sure about it 🤔 
What if `ASYNCAPI_METRICS = 'true'` and `CI = 'true'`? In that situation we would be sending metrics when applying the OR condition. So if that scenario is possible, shouldn't we apply the AND condition instead?

Reply:
You are right 👍